### PR TITLE
output documentation

### PR DESCRIPTION
**Pull request by vbatts**
This carries over the efforts from image-spec and runtime-spec to make
output documentation (PDF and HTML).

Also assumes a version of v0.1.0-dev for now.

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,30 @@
 EPOCH_TEST_COMMIT	:= 91d6d8466e68f1efff7977b63ad6f48e72245e05
 
+DOCKER		?= $(shell command -v docker 2>/dev/null)
+PANDOC		?= $(shell command -v pandoc 2>/dev/null)
+
+OUTPUT_DIRNAME	?= output/
+DOC_FILENAME	?= oci-distribution-spec
+
+PANDOC_CONTAINER ?= docker.io/vbatts/pandoc:1.19.1-3.fc27.x86_64
```
[the remark this answers is not in the document]
Comment (vbatts) [reply Makefile]: that would be nice. I have not investigated an org or team that we could share.

[the remark this answers is not in the document]
Comment (vbatts) [reply Makefile]: I'm trying out https://quay.io/repository/oci/pandoc?tab=tags

**Comment by jzelinskie** [comment]
LGTM

**Comment by vbatts** [comment]
@opencontainers/distribution-spec-maintainers PTAL

File: .tool/genheader.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 The Linux Foundation
```
Comment (mikebrow) [inline .tool/genheader.go]: might want to change the year here..

File: version.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 The Linux Foundation
```
Comment (mikebrow) [inline version.go]: ditto

File: .tool/genheader.go (hunk 2 above)
Comment (vbatts) [reply .tool/genheader.go]: hmmm, fair. It's fairly directly copied code, but I get it.

Comment (mikebrow) [reply .tool/genheader.go]: yeah one way to handle that is to put a year range or just add one for this year..

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**